src/Maestro/SubscriptionActorService/PullRequestActor.cs

```diff
@@ -10,12 +10,12 @@
 using System.Runtime.Serialization;
 using System.Text;
 using System.Threading.Tasks;
+using Castle.Core.Internal;
```

Comment:
Where is this used?

Reply:
I was wondering the same and checked out the branch. Looks like it's for the IsNullOrEmpty Extension Methods used in LNS 748 and 973

Reply:
Would it be better to write our own or use this one? We already use Castle.Core, but this still seems a little out of place.

Reply:
Agreed.

Also this is just replacing `(repoDependencyUpdate.RequiredUpdates == null || !repoDependencyUpdate.RequiredUpdates,Any())`

IMO, that's clearer than `IsNullOrEmpty` against a collection anyway.

Reply:
Updated. Using `.Count() < 1` instead.